packages/ui/src/modal/ConfirmationModal.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import { TranslatedString } from '@bigcommerce/checkout/locale';
+
+import { Button, ButtonSize, ButtonVariant } from '../button';
+
+import Modal from './Modal';
+import ModalHeader from './ModalHeader';
+
+interface ConfirmationModalProps {
+    headerId: string;
+    messageId: string;
+    isModalOpen: boolean;
+    onRequestClose: () => void;
+    action: () => void;
+}
+
+const ConfirmationModal = ({
+    headerId,
+    messageId,
+    isModalOpen,
+    action,
+    onRequestClose,
+}: ConfirmationModalProps) => {
+    return (
+        <Modal
+            additionalModalClassName="modal--confirm"
+            footer={
+                <Button
+                    onClick={action}
+                    size={ButtonSize.Small}
+                    testId="edit-multi-shipping-cart-confirm-button"
```

Comment:
Should this be dynamic from props as well?

Reply:
Good catch, I have removed this `testId` as we are able to assert the modal existence by other methods.